src/main/webapp/app/admin/user-management/user-management.component.ts

```diff
@@ -41,19 +46,47 @@ export class UserManagementComponent implements OnInit, OnDestroy {
         private router: Router,
         private eventManager: JhiEventManager,
     ) {
-        this.itemsPerPage = ITEMS_PER_PAGE;
+        this.itemsPerPage = 50;
```

Comment:
```suggestion
        this.itemsPerPage = ITEMS_PER_PAGE;
```

Reply:
Let's stay with the default value here. I increased this recently and this should be fine (even if it was specified differently in the GitHub issue), except we run into performance problems